app/Http/Client/Alchemy/AlchemyPendingRequest.php

```diff
@@ -241,16 +241,16 @@ public function nftMetadataBatch(Collection $nfts, Network $network): Web3NftsCh
 
         $response = self::post('getNFTMetadataBatch', [
             'tokens' => $tokens,
-        ])->json();
+        ])->json('nfts');
 
         /** @var Collection<int, Nft>  $response */
         $nftItems = collect($response)
-            ->filter(fn ($nft) => $this->filterNft($nft))
+            ->filter(fn ($nft) => $this->filterNftV3($nft))
```

Comment:
I had to keep the old method names because it was used by other methods, that is why implemented new methods with `V3` suffix, otherwise all unrelated test would fail.  I handled this comment here: https://github.com/ArdentHQ/dashbrd/pull/504/files#diff-957272cbfdd990765850dc7924f1ac272f6ea71fb3403acbe633fe548369ebf6R248

Reply:
yeah but that PR is merged into this branch already? so can you not get rid of the remaining v3 references by now? 🤔